### PR TITLE
design diagram with image

### PR DESCRIPTION
**Pull request by namframgia**

### CODE REVIEW

**Comment by namframgia** [comment]
https://edu-redmine.framgia.vn/issues/4642

File: app/Http/Controllers/Admin/DiagramController.php [hunk 1]
```diff
@@ -342,6 +381,21 @@ public function showDesignWithoutDiagram(Request $request, $id)
         return view('test.workspace.design_without_diagram', compact('workspace'));
     }
 
+    public function getLocationsArray($object, $workspace_id)
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/DiagramController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 384</br>Variable "workspace_id" is not in valid camel caps format

File: app/Http/Controllers/Admin/DiagramController.php [hunk 2]
```diff
@@ -342,6 +381,21 @@ public function showDesignWithoutDiagram(Request $request, $id)
...
+    {
+        $keys = [];
+
+        foreach ($object as $key => $val) {
+            $newArray = [];
+            $newArray['name'] =  $key;
+            $newArray['color'] =  $val->color;
+            $newArray['workspace_id'] =  $workspace_id;
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/DiagramController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 392</br>Variable "workspace_id" is not in valid camel caps format

File: app/Repositories/LocationRepository.php [hunk 3]
```diff
@@ -247,4 +247,12 @@ public function getTotalUsersInDay($location, $filter)
 
         return $data;
     }
+
+    public function getLocationIdByWorkspace($workspace_id)
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>**Line**: 251</br>Variable "workspace_id" is not in valid camel caps format

File: app/Repositories/LocationRepository.php [hunk 4]
```diff
@@ -247,4 +247,12 @@ public function getTotalUsersInDay($location, $filter)
...
+    {
+        return $this->model
+            ->where('workspace_id', $workspace_id)
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>**Line**: 254</br>Variable "workspace_id" is not in valid camel caps format

File: app/Models/DesignDiagram.php [hunk 5]
```diff
@@ -27,4 +27,15 @@ public function getDesignDiagramAttribute()
...
     }
+
+    public function getDiagramContentAttribute()
+    {
+        if ($this->content != strip_tags($this->content)) {
+            return $this->content;
+        }
+
+        return '<div class="text-center"><span class="text-danger">'
+                . __('This workspace don\'t have this kind of diagram')
```
Comment (BunnyPi04) [inline app/Models/DesignDiagram.php]: hạn chế return html nhé
có thể sửa được không?

File: resources/views/test/workspace/diagram_image.blade.php [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<div class="col-lg-12 text-center">
+        {!! Html::image($workspace->photo, null, ['usemap' => '#image-map']) !!}
```
Comment (BunnyPi04) [inline resources/views/test/workspace/diagram_image.blade.php:2]: pull sau tab lại đoạn này nhé

**Pull request merged**